modules/xmpp/JingleSession.js

```diff
@@ -190,6 +190,28 @@ export default class JingleSession {
      */
     acceptOffer(jingle, success, failure) {}
 
+    /**
+     * Although it states "replace transport" it does accept full Jingle offer
+     * which should contain new ICE transport details.
+     * @param jingleOfferElem an element Jingle IQ that contains new offer and
+     *        transport info.
+     * @param success callback called when we succeed to accept new offer.
+     * @param failure function(error) called when we fail to accept new offer.
+     */
+     replaceTransport(jingleOfferElem, success, failure){}
+     
+     /**
+     * Will put and execute on the queue a session modify task. Currently it
```

Comment:
Let's modify this comment to be generic. The references for the implementation should be removed. Maybe you can leave it:
```
/**
 * Will put and execute on the queue a session modify task.
 * @param {jQuery} jingleContents - jQuery selector pointing to the jingle element of the session modify IQ. 
 */
```
WDYT?